beta-architecture/diagrams/tdr-beta-high-level-architecture.svg

```diff
@@ -0,0 +1 @@
25l-1-190h47c2 12-1 28 3 38 12-26 28-41 61-42" id="bt"/><g id="j"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#bp"/><use transform="matrix(0.05,0,0,0.05,12.950000000000001,0)" xlink:href="#bo"/><use transform="matrix(0.05,0,0,0.05,23.900000000000002,0)" xlink:href="#be"/><use transform="matrix(0.05,0,0,0.05,39.900000000000006,0)" xlink:href="#bs"/><use transform="matrix(0.05,0,0,0.05,44.900000000000006,0)" xlink:href="#bt"/></g><path d="M182-130c37 4 62 22 62 59C244 23 116-4 24 0v-248c84 5 203-23 205 63 0 31-19 50-47 55zM76-148c40-3 101 13 101-30 0-44-60-28-101-31v61zm0 110c48-3 116 14 116-37 0-48-69-32-116-35v72" id="bu"/><g id="k"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#bl"/><use transform="matrix(0.05,0,0,0.05,12.950000000000001,0)" xlink:href="#bu"/></g><path d="M24-248c93 1 206-16 204 79-1 75-69 88-152 82V0H24v-248zm52 121c47 0 100 7 100-41 0-47-54-39-100-39v80" id="bv"/><path d="M195-6C206 82 75 100 31 46c-4-6-6-13-8-21l49-6c3 16 16 24 34 25 40 0 42-37 40-79-11 22-30 35-61 35-53 0-70-43-70-97 0-56 18-96 73-97 30 0 46 14 59 34l2-30h47zm-90-29c32 0 41-27 41-63 0-35-9-62-40-62-32 0-39 29-40 63 0 36 9 62 39 62" id="bw"/><g id="l"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#bv"/><use transform="matrix(0.05,0,0,0.05,12,0)" xlink:href="#bd"/><use transform="matrix(0.05,0,0,0.05,22.950000000000003,0)" xlink:href="#bg"/><use transform="matrix(0.05,0,0,0.05,32.95,0)" xlink:href="#bh"/><use transform="matrix(0.05,0,0,0.05,38.900000000000006,0)" xlink:href="#bw"/><use transform="matrix(0.05,0,0,0.05,49.85000000000001,0)" xlink:href="#ba"/><use transform="matrix(0.05,0,0,0.05,56.85,0)" xlink:href="#aZ"/><use transform="matrix(0.05,0,0,0.05,66.85000000000001,0)" xlink:href="#bg"/></g><path d="M135-194c53 0 70 44 70 98 0 56-19 98-73 100-31 1-45-17-59-34 3 33 2 69 2 105H25l-1-265h48c2 10 0 23 3 31 11-24 29-35 60-35zM114-30c33 0 39-31 40-66 0-38-9-64-40-64-56 0-55 130 0 130" id="bx"/><g id="m"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#br"/><use transform="matrix(0.05,0,0,0.05,12.950000000000001,0)" xlink:href="#bx"/><use transform="matrix(0.05,0,0,0.05,23.900000000000002,0)" xlink:href="#bi"/><use transform="matrix(0.05,0,0,0.05,28.900000000000002,0)" xlink:href="#bd"/><use transform="matrix(0.05,0,0,0.05,39.85,0)" xlink:href="#bb"/><use transform="matrix(0.05,0,0,0.05,49.85,0)" xlink:href="#bo"/></g><path d="M4 7l51-268h42L46 7H4" id="by"/><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#by" id="n"/><path d="M123 10C108 53 80 86 19 72V37c35 8 53-11 59-39L3-190h52l48 148c12-52 28-100 44-148h51" id="bz"/><g id="o"><use transform="matrix(0.05,0,0,0.05,0,0)" xlink:href="#bl"/><use transform="matrix(0.05,0,0,0.05,12.950000000000001,0)" xlink:href="#bs"/><use transform="matrix(0.05,0,0,0.05,17.950000000000003,0)" xlink:href="#ba"/><use transform="matrix(0.05,0,0,0.05,24.950000000000003,0)" xlink:href="#bh"/><use transform="matrix(0.05,0,0,0.05,30.900000000000002,0)" xlink:href="#bz"/></g><g id="p"><use 
```

Comment:
Just noticed that the Monitoring box includes Splunk, which we're probably not going to use. We'll almost certainly be using CloudWatch metrics, and probably Grafana as well.

Reply:
Resolved